src/migration/migration.cpp

```diff
@@ -50,7 +50,7 @@ void Migration::ApplyMigration(const std::string& migrationScript)
     std::ifstream file(path);
 
     if (!file.is_open()) {
-        AOS_ERROR_THROW("failed to open migration script: " + migrationScript, aos::ErrorEnum::eRuntime);
+        AOS_ERROR_THROW(aos::ErrorEnum::eRuntime, "failed to open migration script: " + migrationScript);
```

Comment:
redundant namespace usage
 ```suggestion
        AOS_ERROR_THROW(ErrorEnum::eRuntime, "failed to open migration script: " + migrationScript);
```

Reply:
Done